src/include/libpmemfile-core.h

```diff
@@ -60,12 +60,19 @@ void pmemfile_pool_close(PMEMfilepool *pfp);
 
 PMEMfile *pmemfile_open(PMEMfilepool *pfp, const char *pathname, int flags,
 		...);
+PMEMfile *pmemfile_openat(PMEMfilepool *pfp, PMEMfile *file,
```

Comment:
My general question is why are we using a PMEMfile * as 2nd parameter instead of the fd?